src_new/util.js

```diff
@@ -1102,6 +1102,41 @@ exports.getAdUnitConfig = function(sizes, currentSlot){
 			return Object.keys(slotConfig['config']).toString().toLowerCase().indexOf(kgpv.toLowerCase()) > -1 ? true : false;
 		}
 	}
+	// checks if regex is present and enabled
+	function isregexEnabled() {
+		return slotConfig && (refThis.isOwnProperty(slotConfig, CONSTANTS.COMMON.MCONF_REGEX)) && (slotConfig[CONSTANTS.COMMON.MCONF_REGEX] == true) ? true : false;
```

Comment:
Instead can you just write and check below line if it dose same as line no 1107
`return slotConfig && slotConfig[CONSTANTS.COMMON.MCONF_REGEX]`

Reply:
Thank You! 
checked as per the comment, respectively made the change!